imports/plugins/custom/top-products/client/templates/topProducts.html

```diff
@@ -1,24 +1,21 @@
 <template name="topProducts">
 
-    <div class="flex flex-column flex-align-center flex-2x comp-order container-lg">
+  <div class="flex flex-column flex-align-center flex-2x comp-order container-lg">
 
-      <div class="flex-item">
-        <div class="panel panel-default user-orders-list">
-          <div class="panel-heading">
+    <div class="flex-item">
```

Comment:
This doesn't seem to be related to the feature you are currently working.
Can you color more on this?

Reply:
Thanks for noting this. I had to implement the feedback received on the top products analysis feature while working on the improved media gallery feature.